src/main/java/Duke/java/tasks.java

```diff
@@ -0,0 +1,40 @@
+package Duke.java;
+
+public class tasks {
+    protected String task;
+    protected boolean done;
+    protected static int taskNum=0;
+
+    public tasks() {
+        this.done=false;
```

Comment:
`Boolean` values should be named to sound like `boolean`(e.g. `isSet`, `isDone`)